blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -439,7 +443,7 @@ export const actions = {
                                 return { ...data,
                                     id: event.jenkins_object_id, // make sure the runId is set so we can find it later
                                     state: 'RUNNING', // This is a horrible hack due to issues with QUEUED status
-                                    result: event.job_run_status,
+                                    result: 'UNKNOWN',
```

Comment:
Ah, ignore my last comment. I was mixing up `state` and `result`. However, I see the state is set to `RUNNING` with an interestingly teasing, yet ultimately useless comment. Was prob me .... bad boy. It's be nice to know what the "issues with QUEUED status" were.